### PR TITLE
Update weights

### PR DESCRIPTION
**Pull request by adamdossa**
Updates the weights for Polymesh v5.4

### CODE REVIEW

File: pallets/weights/src/pallet_compliance_manager.rs [hunk 1]
```diff
@@ -57,17 +57,13 @@ impl pallet_compliance_manager::WeightInfo for SubstrateWeight {
...
-        Weight::from_ref_time(8_114_000)
-            // Standard Error: 39_365
-            .saturating_add(Weight::from_ref_time(2_949_096).saturating_mul(a.into()))
-            // Standard Error: 39_365
-            .saturating_add(Weight::from_ref_time(411_324).saturating_mul(b.into()))
-            // Standard Error: 39_365
-            .saturating_add(Weight::from_ref_time(671_782).saturating_mul(c.into()))
-            // Manually set for `d`
-            .saturating_add(Weight::from_ref_time(721_394).saturating_mul(d.into()))
+    fn condition_costs(a: u32, _b: u32, c: u32, _d: u32) -> Weight {
```
Comment (Neopallium) [inline pallets/weights/src/pallet_compliance_manager.rs]: Shouldn't be ignoring `b` here.

File: pallets/weights/src/pallet_compliance_manager.rs [hunk 2]
```diff
@@ -150,11 +146,9 @@ impl pallet_compliance_manager::WeightInfo for SubstrateWeight {
     // Storage: ComplianceManager AssetCompliances (r:1 w:1)
     // Storage: ComplianceManager TrustedClaimIssuer (r:1 w:0)
     /// The range of component `c` is `[1, 6]`.
-    fn change_compliance_requirement(c: u32) -> Weight {
-        // Minimum execution time: 55_072 nanoseconds.
-        Weight::from_ref_time(57_468_205)
-            // Standard Error: 32_981
-            .saturating_add(Weight::from_ref_time(683_464).saturating_mul(c.into()))
+    fn change_compliance_requirement(_c: u32) -> Weight {
```
Comment (Neopallium) [inline pallets/weights/src/pallet_compliance_manager.rs]: Shouldn't ignore `c` here.

File: pallets/weights/src/pallet_corporate_actions.rs [hunk 3]
```diff
@@ -117,13 +117,11 @@ impl pallet_corporate_actions::WeightInfo for SubstrateWeight {
...
     /// The range of component `w` is `[0, 1000]`.
     /// The range of component `t` is `[0, 500]`.
-    fn initiate_corporate_action_use_defaults(w: u32, t: u32) -> Weight {
-        // Minimum execution time: 218_163 nanoseconds.
-        Weight::from_ref_time(192_138_698)
-            // Standard Error: 2_478
-            .saturating_add(Weight::from_ref_time(181_016).saturating_mul(w.into()))
-            // Standard Error: 4_953
-            .saturating_add(Weight::from_ref_time(24_264).saturating_mul(t.into()))
+    fn initiate_corporate_action_use_defaults(w: u32, _t: u32) -> Weight {
```
Comment (Neopallium) [inline pallets/weights/src/pallet_corporate_actions.rs]: Param `t` is important, number of target identities.

File: pallets/weights/src/pallet_identity.rs [hunk 4]
```diff
@@ -188,47 +188,43 @@ impl pallet_identity::WeightInfo for SubstrateWeight {
...
-        Weight::from_ref_time(0)
-            // Manually set for `a`
-            .saturating_add(Weight::from_ref_time(163_000).saturating_mul(a.into()))
-            // Manually set for `p`
-            .saturating_add(Weight::from_ref_time(296_000).saturating_mul(p.into()))
-            // Standard Error: 140_388
-            .saturating_add(Weight::from_ref_time(33_906_000).saturating_mul(l.into()))
-            // Standard Error: 140_388
-            .saturating_add(Weight::from_ref_time(33_316_000).saturating_mul(e.into()))
+    fn permissions_cost(_a: u32, _p: u32, l: u32, e: u32) -> Weight {
```
Comment (Neopallium) [inline pallets/weights/src/pallet_identity.rs]: Can't ignore these parameters.

File: pallets/weights/src/pallet_staking.rs [hunk 5]
```diff
@@ -341,12 +341,9 @@ impl pallet_staking::WeightInfo for SubstrateWeight {
...
     /// The range of component `l` is `[1, 32]`.
     fn rebond(u: u32) -> Weight {
-        // Minimum execution time: 47_949 nanoseconds.
-        Weight::from_ref_time(49_739_043)
-            // Standard Error: 5_104
+        // Minimum execution time: 103_392 nanoseconds.
+        Weight::from_ref_time(151_335_738)
             .saturating_add(Weight::from_ref_time(1_000).saturating_mul(u.into()))
-            .saturating_add(DbWeight::get().reads(4))
-            .saturating_add(DbWeight::get().writes(3))
```
Comment (Neopallium) [inline pallets/weights/src/pallet_staking.rs]: These reads and writes shouldn't have been removed.

**Review by Neopallium — CHANGES_REQUESTED** [review]

**Review by HenriqueNogara — APPROVED** [review]

File: pallets/weights/src/pallet_compliance_manager.rs (hunk 1 above)
Comment (HenriqueNogara) [reply pallets/weights/src/pallet_compliance_manager.rs]: ~~Is this benchmark being used?~~ Never mind. It's called in combination with other benchmarks

**Review by Neopallium — APPROVED** [review]

**Pull request merged**